README.md

```diff
@@ -1,110 +1,124 @@
 # The missing introduction to APCA
 
-## What is APCA?
+## _What is APCA in three sentences:_
+It is a supra threshold visual contrast prediction model that is specifically tuned to readability on self-illuminated displays. It follows human perception, including critical aspects such as spatial frequency, meaning font weight and size. The APCA guidelines also inform various use cases, such as body text, fluent text, spot reading text, and non-text items.
```

Comment:
Terms like "supra threshold" should be explained on first use.

Reply:
It is a common term, though possibly not plain language. A descriptor should be a footnote.